### ISSUE TITLE
240p? videos

### ISSUE THREAD

**Issue by vol-2** [mps-youtube/mps-youtube#1041]
want to watch 240p videos. sending them to an old CRT. this used to work, but doesn't seem to anymore.
Now it says maxres is 360. many YouTube videos are still 240p.

**Comment by AhmedMohamed365** [comment]
is this problem fixed now or what ?

**Comment by vol-2** [comment]
I don't think so. I never heard any comment about it so I assume that whoever maintains this doesn't care about my issue and probably doesn't consider it an issue at all.

My particular case is that I'm using a raspberry pi on a 240p CRT for retrogaming. I have multiple images that use Raspbian, and it's great to have the ability to play old TV footage via YouTube on the old CRT. I actually used to be able to do this with mps-youtube, but the functionality broke at some point over the last couple years. It seems to be some kind of generic error that occurs when the system is looking for a specific video mode.
It's the error I get when trying to open dosbox in 240p without some scaling method. (because dosbox console is 640*400)
So whatever changed has to do with the resolution that mps-youtube is expecting as a minimum, and it appears 320*240 is no longer a valid option.

**Comment by bydioeds** [comment]
Same issue, i suposse is never going to be solved
